cli/cmd/project/project.go

```diff
@@ -72,6 +72,6 @@ func inferProjectName(ctx context.Context, adminClient *client.Client, org, path
 type project struct {
 	Name         string `header:"name" json:"name"`
 	Public       bool   `header:"public" json:"public"`
-	GithubURL    string `header:"github" json:"github"`
+	GithubURL    string `header:"git url" json:"github"`
```

Comment:
I know this was suggested in the bug bash, but IMO this should be "github" – the URLs we return here are *not* valid Git URLs (they don't end in `.git`)

Reply:
Yes, see my comment there. It does not generalize to other platforms than Github anyway, so we should keep the old title